lib/util/cache/package/redis.ts

```diff
@@ -5,9 +5,10 @@ import { logger } from '../../../logger';
 import { compress, decompress } from '../../compress';
 
 let client: ReturnType<typeof createClient> | undefined;
+let rprefix: string | undefined;
 
 function getKey(namespace: string, key: string): string {
-  return `${namespace}-${key}`;
+  return `${rprefix}:${namespace}-${key}`;
```

Comment:
```suggestion
  return `${rprefix}${namespace}-${key}`;
```

If you need a colon then add it as part of redisPrefix